crates/bevy_gltf/src/loader.rs

```diff
@@ -518,15 +518,21 @@ async fn load_gltf<'a, 'b>(
     Ok(())
 }
 
+fn node_name(node: &Node) -> Name {
+    let name = node
+        .name()
+        .map(|s| s.to_string())
+        .unwrap_or_else(|| format!("bevy-gltf-node-{}", node.index()));
```

Comment:
Imo we can omit the bevy prefix here, as that context is implicit. When these names start showing up in scene editors / viewers, a bunch of `bevy-` prefixes will feel clunky (in my opinion).

Names also feel like "proper nouns" and both blender and godot use PascalCase for these names by convention. I think my vote is for `GltfNodeX` as the pattern here.

Reply:
done 👍